content/Voice Over - Fernanda Toledo.md

```diff
@@ -0,0 +1,63 @@
+# How to make your app accessible using Voice Over
+
+![](https://github.com/rootstrap/blog/blob/voice-over/content/images/glasses.jpg)
+
+In a world where everything is relying more and more on technology, more tools are being provided for us developers to make our apps more accessible to everyone. Having limited sight should not be an impediment to use any app. As developers or designers, we need to raise awareness of all of those we're leaving behind and take a little time to make sure that our apps are accessible to everyone. Everyone wins, you'll broaden your market and increase your exposure, which will always lead to a positive impact on your profits. And the most important, by including people with disabilities you'll make a huge difference in their lives!
+
+So, what makes an app accessible? According to the [Web Accessible Initiative](https://www.w3.org/WAI/fundamentals/accessibility-intro/), it means that the app has been designed so that people with disabilities are capable of using it, meaning they can not only perceive and understand it but also navigate and interact with the app. With that said, how can we make our apps accessible? There is a broad set of tools to help users with disabilities, we need to take action and remove the accessibility barriers by providing alternative solutions with the help of these tools. Specifically, in this article we'll talk about how to use, test, and improve accessibility using Voice Over.
```

Comment:
```suggestion
So, what makes an app accessible? According to the [Web Accessible Initiative](https://www.w3.org/WAI/fundamentals/accessibility-intro/), it means that the app has been designed so that people with disabilities are capable of using it, meaning they can not only perceive and understand it but also navigate and interact with the app. With that said, how can we make our apps accessible? There is a broad set of tools to help users with disabilities, we need to take action and remove the accessibility barriers by providing alternative solutions with the help of these tools. In this article, we'll talk about how to use, test, and improve accessibility using VoiceOver.
```